### PR TITLE
Add missing action/upgrade.go license

### PR DESCRIPTION
**Pull request by Itxaka**
license checker was failing due merging 2 prs at the same
time, one that dixed all license headers and the other
that introduced this file, so the license checker was
green until now :D

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

**Review by dragonchaser — APPROVED** [review]

File: pkg/action/upgrade.go [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/*
+Copyright The Helm Authors, SUSE LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (mattfarina) [inline pkg/action/upgrade.go:8]: The URL should not be indented. We have it all the way on the left in the other files.

**Review by mattfarina — CHANGES_REQUESTED** [review]
Good catch on the missing header. One minor nit...

File: pkg/action/upgrade.go (hunk 1 above)
Comment (Itxaka) [reply pkg/action/upgrade.go:8]: we dont because I copied it from https://github.com/rancher-sandbox/hypper/blob/main/pkg/action/install.go#L8 lol

Comment (viccuad) [reply pkg/action/upgrade.go:8]: It seems that some one them are indented https://github.com/rancher-sandbox/hypper/blob/main/pkg/action/install_test.go#L8
and some of them are not https://github.com/rancher-sandbox/hypper/blob/main/pkg/action/action.go#L7

**Review by viccuad — APPROVED** [review]

**Pull request merged**